be/src/storage/vectorized/chunk_aggregator.h

```diff
@@ -70,6 +70,7 @@ class ChunkAggregator {
 
     uint32_t _reserve_rows;
 
+    // total limit of aggregate rows
```

Comment:
FYI: could you please give this variable another name, it's too similar to your newly added variable.

Reply:
Agree with you

Reply:
ok